### PR TITLE
chore(www): convert ImagePlaceholder used in guidelines to theme-ui

### PR DESCRIPTION
**Pull request by abhishekjakhar**
## Description

Convert the `<ImagePlaceholder/>` component to theme-ui

### Screenshot(s)

image-placeholder-1
image-placeholder-2

### How to test

The ImagePlaceholder is being used only on typography page, it is used at 3 places, the images are provided above

The images provided above should look same on both local and production URL mentioned below

**Local URL:** http://localhost:8000/guidelines/typography/
**Production URL:** https://www.gatsbyjs.org/guidelines/typography/

### CODE REVIEW

**Comment by abhishekjakhar** [comment]
@tesseralis 
**Gatsby Build Service - using-styled-components** this is failing for every build, I don't have any idea about this

File: www/src/components/guidelines/image-placeholder.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
-import React from "react"
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Box } from "theme-ui"
```
Comment (tesseralis) [inline www/src/components/guidelines/image-placeholder.js:3]: Can we combine these together or does it break the JSX pragma?

```suggestion
import { jsx, Box } from "theme-ui"
```

**Comment by abhishekjakhar** [comment]
> lgtm, just a minor comment about whether we can combine imports

Yes, I combined them

**Comment by abhishekjakhar** [comment]
@tesseralis can I get a review on this and my other PR’s?

**Pull request merged**